src/window/gui/GuiMenuBar.h

```diff
@@ -7,5 +7,6 @@ namespace LUS {
 class GuiMenuBar : public GuiElement {
   public:
     using GuiElement::GuiElement;
+    float MenuBarHeight;
```

Comment:
I don't think we want to store this on the class. Can you remove this part and just hard code it where you currently use this?